index.d.ts

```diff
@@ -0,0 +1,31 @@
+declare module 'resin-jwt' {
+	import Strategy from 'passport-jwt';
+
+	interface StrategyOpts {
```

Comment:
I think it's common to spell this thing as `SomethingOptions`, but not that important.